src/printing.py

```diff
@@ -11,7 +11,11 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+rounded_y = round(y, 2)
+print("x is %s, y is %s, z is \"%s\"" % (x, rounded_y, z))
 
 # Use the 'format' string method to print the same thing
+print("x is {0}, y is {1}, z is \"{2}\"".format(x, rounded_y, z))
```

Comment:
Same as above, check out this link for how to round with f-strings https://stackoverflow.com/questions/45310254/fixed-digits-after-decimal-with-f-strings